### ISSUE TITLE
Different client and server versions when installing docker on ubuntu (for the first time). 

### ISSUE THREAD

**Issue by mappelgren** [docker/docker#14468]
Hi,
I just tried to install docker, using the installation guidelines given on your homepage. When trying to run the hello world test script I was given this error:
`Error response from daemon: client and server don't have same version (client : 1.19, server: 1.18)`

I realise that there is a known issue with having a higher version client than server, but you might not be aware that it happens straight after install, at least for me. 

`docker version:`
`Client version: 1.7.0
Client API version: 1.19
Go version (client): go1.4.2
Git commit (client): 0baf609
OS/Arch (client): linux/amd64
Error response from daemon: client and server don't have same version (client : 1.19, server: 1.18)`

`docker info:`
`Error response from daemon: client and server don't have same version (client : 1.19, server: 1.18)`

`uname -a:`
`Linux uwp-cookie-machine 3.13.0-55-generic #92-Ubuntu SMP Sun Jun 14 18:32:20 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux`

How reproducible:
I haven't tried reinstalling docker, I'm not invested enough to find out the best way of doing this. 

Steps to reproduce:
1. Install docker:     `wget -qO- https://get.docker.com/ | sh`
2. Run hello world test script:    `docker run hello-world`

**Comment by cpuguy83** [comment]
Hmmm, something is not right here.
The client and the daemon are distributed as a single binary, as such they use the same API version number.

**Comment by mappelgren** [comment]
Maybe I just did something weird when I was doing it, or I dont know. Is there any best practice for reinstalling it? I can have a go and see if I just messed something up.

**Comment by cpuguy83** [comment]
@mappelgren It seems like the client bin you are executing is different from the one being used to fire up the daemon... so maybe you have something in your `$PATH`

**Comment by mappelgren** [comment]
@cpuguy83 I looked through my path and I only found docker at /usr/bin/docker which is the same as I get when I run which docker. Is that what I'm looking for? Another docker in my $PATH, or would it be called something else?

**Comment by jfrazelle** [comment]
try a `find / -name "docker"` and see what shows up

**Comment by andypohl** [comment]
I get the same issue in Windows 7.  I downloaded Boot2Docker within the last few days.  When I double-click Boot2Docker Start, it seems to start up fine.  It errors and says I have client 1.19 and server 1.18 when I execute "docker version" or any other docker command.  When I ssh into the VM with "boot2docker ssh", things work.  In that case, I have client 1.18.  I would prefer to run docker outside of the VM though.

**Comment by andypohl** [comment]
Well I finally figured it out.  My boot2docker.iso was still 1.6.2.  I don't know why.  I deleted it many times and I thought I'd be getting the latest one with "boot2docker init".  Finally I downloaded the latest .iso and used the --iso option with boot2docker init.  I think everything's ok now, but that was pretty frustrating.  I'd be really surprised if the all-in-one Windows installer included an old iso.  I have other problems with boot2docker (e.g. boot2docker download/upgrade), but I think I can get around everything now.  As long as I have a way of running docker in Windows without launching a full-blown VM, I'll bear with it.
